### DISCUSSION TITLE
Performance tests

### DISCUSSION THREAD

**Discussion by avidseeker** [privacytests/privacytests.org#214] [Ideas]
This could be a separate project, but it would be nice to have a table of performance tests for different browsers. https://browserbench.org/ could be useful for this task.

**Comment by avidseeker** [comment]
Relevant benchmarks: https://arewefastyet.com/win10/benchmarks/overview?numDays=60

**Comment by zkisaboss** [comment]
What about the tests for privacy that this uses? It's unrelated but is there a way for me to test my hardened firefox to see if the settings are working?

**Reply by avidseeker** [reply]
Possibly https://deviceinfo.me
